src/shell/scripts/omp.ps1

```diff
@@ -26,7 +26,7 @@ function global:Start-Utf8Process {
 
 $env:POWERLINE_COMMAND = "oh-my-posh"
 $env:CONDA_PROMPT_MODIFIER = $false
-$env:SHELL_VERSION = $PSVersionTable.PSVersion.ToString()
+$global:OMP_PSVERSION = $PSVersionTable.PSVersion.ToString()
```

Comment:
OK, fair point. I'm the env var person 😆

Reply:
Aha, this is just in case the value of env `SHELL VERSION` is inherited by subprocesses that coincidentally depend on it. So leave it in PowerShell can be a better choice. 😆